sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/BaseCrossDriver.java

```diff
@@ -0,0 +1,73 @@
+package com.saucelabs.appium;
+
+import io.appium.java_client.AppiumDriver;
+import io.appium.java_client.MobileBy;
+import io.appium.java_client.android.AndroidDriver;
+import io.appium.java_client.ios.IOSDriver;
+import io.appium.java_client.service.local.AppiumDriverLocalService;
+import io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException;
+import org.junit.After;
+import org.junit.Before;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Created by saikrisv on 01/06/17.
+ */
+public class BaseCrossDriver {
+    public AppiumDriver driver;
+    private static AppiumDriverLocalService service;
+
+    @Before
+    public void setUp() throws Exception {
+        service = AppiumDriverLocalService.buildDefaultService();
```

Comment:
do we really want the sample code to be starting appium for us? most of the languages have appium started separately. but if this is the way people tend to do it in JAVA, I'm ok with that. we should then have a readme explaining how to run it